workflow-job

```diff
@@ -274,6 +273,10 @@ def findImapPortCmd(dockerContainer) {
     return "export JAMES_PORT=`docker inspect -f '{{(index (index .NetworkSettings.Ports \"143/tcp\") 0).HostPort}}' ${dockerContainer}`"
 }
 
+def containerIp(dockerContainer) {
+    return "export JAMES_IP=`docker inspect -f '{{(index .NetworkSettings.Networks.platform_default.IPAddress)}}' ${dockerContainer}`" 
```

Comment:
if you export it, you will redefine it each time you call it.

Reply:
JAMES_IP is a strange name given the name of the function

Reply:
> JAMES_IP is a strange name given the name of the function

That is what it is: James IP address...